src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing the Meshery Adapter for Open Service Mesh"
+subtitle: ""
+date: 2020-08-20 10:17:05 +0000
+author: Lee Calcote
+thumbnail: ./meshery-open-service.png
+tags:
+ - Community
+ - Projects
+ - News
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import conformance from "./conformance-results.png";
+import meshery from "./meshery-open-service.png"
+
+<BlogWrapper>
+
+### Adding another service mesh to the landscape and another adapter to Meshery
```

Comment:
This is to be added as a subtitle field in frontmatter.

Reply:
@lolyparty as you have already added it to the subtitle, it's no longer required here. It's duplicate. Please remove this.